tests/projects/FunctionAppIsolated/.gitignore

```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and

```

Comment:
Why is this file needed?

Reply:
The whole FunctionAppIsolated project was generated with Visual Studio, and the gitignore file as well. I don't think it's needed, and it can be removed.